GoogleForCreators/web-stories-wp
# Add contentType filter attribute to media3pApiProvider

## Summary

Add the contentType filter attribute to listMedia(), and fold listCategoryMedia into listMedia.

## Relevant Technical Choices

listCategoryMedia has been folded into listMedia with a 'filter' attribute, so that the filter attributes are listed in one place.

## To-do

Pass 'contentType' from the provider json configuration to useFetchMediaEffect().

## User-facing changes

N/A

## Testing Instructions

* Apply a search term.
* Apply a category filter.
* Remove the category filter to verify the search term is still taking effect.
* Remove the search term to verify the search term no longer takes effect.
* Check for console errors.

---



Addresses #4210
